RELEASE_NOTES.md

```diff
@@ -1,5 +1,118 @@
+
+#### Improved Reporting to Automate
+
+Chef Infra Client run reporting to Automate now respects attribute `allowlist` and `denylist` configurations set in the `client.rb` configuration. This change allows users to limit the data sent to their Automate servers to prevent indexing sensitive data, or to reduce the necessary storage space on the Automate server.
+
+### Updated Resources
+
+#### homebrew_path
+
+The `homebrew_path` now passed the `homebrew_path` when creating or deleting taps. This change prevents failures when running homebrew in a non-standard location or on a M1 system. Thanks [@mattlqx](https://github.com/mattlqx)!
+
+#### hostname
+
+The `hostname` resource now sets the hostname on Windows systems using native PowerShell calls for increased reliability and allows changing the hostname on domain-attached systems. To change the hostname on a domain-attached system pass a domain administrator account using the new `domain_user` and `domain_password` properties.
+
+#### openssl_x509_certificate
+
+The `openssl_x509_certificate` no longer marks the creation of the X509 certificate file as sensitive since this makes troubleshooting difficult and this content is not sensitive. Thanks [@jasonwbarnett](https://github.com/jasonwbarnett)!
+
+#### windows_firewall_rule
+
+The `windows_firewall_rule` resource now allows specifying multiple IP addresses in the `remote_address` property.
+
+#### windows_pagefile
+
+The `windows_pagefile` resource features improved performance and support for the latest releases of Windows 10. The behavior of the resource has also been improved to make managing pagefiles more predictable:
+
+The `path` property now accepts a drive letter where the pagefile should be stored in addition to the full path the pagefile on disk. For example `C`, `C:`, or `C:\` can now be used to specify a pagefile stored at `C:\pagefile.sys`.
+
+Creating a new pagefile will no longer disable the system managed pagefile by default. If you wish to create a pagefile while also disabled the system managed pagefile set `system_managed false`.
+
+#### windows_printer
+
+The `windows_printer` resource includes improved logging when adding or removing printers.
+
+#### windows_printer_port
+
+The `windows_printer_port` resource has been refactored with several improvements:
+
+- Improved performance when adding and removing ports
+- The ability to update existing ports with new values
+- Improved logging of changes made to ports
+- Deprecated the `description` property, which was not actually setting a description on the ports.
+
+#### windows_security_policy
+
+The `windows_security_policy` resource now limits the value of `ResetLockoutCount` to any value less than that of `LockoutDuration` rather than limiting it to 30 minutes.
```

Comment:
```suggestion
The `windows_security_policy` resource now limits the value of `ResetLockoutCount` to any value less than `LockoutDuration` rather than limiting it to 30 minutes.
```